trade-finance-manager-api/src/v1/rest-mappings/mappings/facilities/mapFacilitySnapshot.test.js

```diff
@@ -141,4 +142,14 @@ describe('mapFacility', () => {
 
     expect(result).toEqual(expected);
   });
+
+  describe('when a tfm facilityStage exists', () => {
```

Comment:
I think we can just add the call to the main test above as it's not really conditional in the main function, it's just calling `mapBssEwcsFacilityStage` - `mapBssEwcsFacilityStage` handles if `facilityStage` exists or not and we're just consuming that here

Reply:
Sure, have removed this separate test👍